R/object.R

```diff
@@ -24,6 +24,18 @@ new_object <- function(.data = NULL, ...) {
       call. = FALSE
     )
   }
+  missing_props <- setdiff(setdiff(prop_names_static(obj_cls), prop_names(.data)), nms)
```

Comment:
Not sure if this is any clearer or not, but the nested setdiff does feel weird to me.

```suggestion
  missing_props <- setdiff(prop_names_static(obj_cls), c(prop_names(.data), nms))
```